src/include/sof/platform.h

```diff
@@ -54,6 +54,14 @@ int platform_context_save(struct sof *sof);
 
 /** @}*/
 
+#ifdef CONFIG_IMX8ULP
+#define convert_host_to_local(addr) (SDRAM0_BASE + ((addr) - MEM_RESERVED))
```

Comment:
can we use ```host_to_local()``` and this aligns the naming with ```cache_to_uncache()```

Reply:
@lgirdwood , I moved this defintion to memory.h, can you please review it?

Reply:
@TangleZ please be patient :) it is usually common to allow 2-3 days for review.